server/src/lib/model/achievements.ts

```diff
@@ -119,7 +120,8 @@ export const hasEarnedBonus = async (
   client_id: string,
   challenge: ChallengeToken
 ) => {
-  const [[{ earned }]] = await db.query(
+  // do not destructure to [[{earned}]], just in case null values returned, in scenarios like challenge=null
+  const [[res]] = await db.query(
```

Comment:
This is fixed in https://github.com/mozilla/voice-web/pull/2373, can be safely removed.

Reply:
omg I'm so annoyed that this comment was sitting in a pull request all along 😅

Reply:
Yeah.